packages/printable-itinerary/src/index.js

```diff
@@ -0,0 +1,225 @@
+  /** Contains OTP configuration details. */
+  config: configType.isRequired,
+  /** Itinerary that the user has selected to view, contains multiple legs */
+  itinerary: itineraryType.isRequired,
+  /** A component class that is used to render icons for legs of an itinerary */
+  LegIcon: PropTypes.elementType.isRequired,
+  /** Contains the preferred format string for time display and a timezone offset */
+  timeOptions: timeOptionsType
+};
+
+PrintableItinerary.defaultProps = {
+  className: null,
+  timeOptions: null
+};
+
+function TransitLeg({ leg, LegIcon, interlineFollows, timeOptions }) {
+  // Handle case of transit leg interlined w/ previous
+  if (leg.interlineWithPreviousLeg) {
+    return (
+      <Styled.CollapsedTop>
+        <Styled.LegBody>
+          <Styled.LegHeader>
+            Continues as{" "}
+            <b>
+              {leg.routeShortName} {leg.routeLongName}
+            </b>{" "}
+            to <b>{leg.to.name}</b>
+          </Styled.LegHeader>
+          <Styled.LegDetails>
+            <Styled.LegDetail>
+              Get off at <b>{leg.to.name}</b> at{" "}
+              {formatTime(leg.endTime, timeOptions)}
+            </Styled.LegDetail>
+          </Styled.LegDetails>
+        </Styled.LegBody>
+      </Styled.CollapsedTop>
+    );
+  }
+
+  return (
+    <Styled.Leg>
+      <Styled.ModeIcon>
+        <LegIcon leg={leg} />
+      </Styled.ModeIcon>
+      <Styled.LegBody>
+        <Styled.LegHeader>
+          <b>
+            {leg.routeShortName} {leg.routeLongName}
+          </b>{" "}
+          to <b>{leg.to.name}</b>
+        </Styled.LegHeader>
+        <Styled.LegDetails>
+          <Styled.LegDetail>
+            Board at <b>{leg.from.name}</b> at{" "}
+            {formatTime(leg.startTime, timeOptions)}
+          </Styled.LegDetail>
+          <Styled.LegDetail>
+            {interlineFollows ? (
+              <span>
+                Stay on board at <b>{leg.to.name}</b>
+              </span>
+            ) : (
+              <span>
+                Get off at <b>{leg.to.name}</b> at{" "}
+                {formatTime(leg.endTime, timeOptions)}
+              </span>
+            )}
+          </Styled.LegDetail>
+        </Styled.LegDetails>
+      </Styled.LegBody>
+    </Styled.Leg>
+  );
+}
+
+TransitLeg.propTypes = {
+  interlineFollows: PropTypes.bool,
+  leg: legType.isRequired,
+  LegIcon: PropTypes.elementType.isRequired,
+  timeOptions: timeOptionsType
+};
+
+TransitLeg.defaultProps = {
+  interlineFollows: false,
+  timeOptions: null
+};
+
+function AccessLeg({ config, leg, LegIcon }) {
+  return (
+    <Styled.Leg>
+      <Styled.ModeIcon>
+        <LegIcon leg={leg} />
+      </Styled.ModeIcon>
+      <Styled.LegBody>
+        <Styled.LegHeader>
+          {getLegModeLabel(leg)}{" "}
+          {leg.distance > 0 && (
+            <span> {humanizeDistanceString(leg.distance)}</span>
+          )}
+          {` to ${getPlaceName(leg.to, config.companies)}`}
```

Comment:
The place name should be bold according to #38.

Reply:
Good catch